src/UI/DataHTMLTooltip.js

```diff
@@ -10,6 +10,7 @@ class DataHTMLTooltip extends HTMLTooltip {
      */
     render (config, items, callbacks) {
         const {wrapperClass, css} = this.options
+        const isTopAutofill = wrapperClass?.includes('top-autofill')
```

Comment:
NAB: Quick and effective way to solve this issue 👍 If we're looking to use this check more in the future, we'd probably want to pass it through in `this.options` and remove relying on this implementation detail.